tableauserverclient/server/request_options.py

```diff
@@ -287,6 +309,15 @@ def get_query_params(self):
         if self.max_age != -1:
             params["maxAge"] = self.max_age
 
+        if (self.viz_height is None) ^ (self.viz_width is None):
```

Comment:
I think this operator is unusual enough that a comment would be helpful

Reply:
Added comment.